### PR TITLE
Add install instructions for linux

### PR DESCRIPTION
**Pull request by berlintam**
:wave: 

I ran into a few problems getting this going on my computer, & thought I'd add a few hints for us Linux people.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,19 +20,33 @@ SignDict uses Elixir and Phoenix. Information on how
...
 
+### Linux instructions:
+
+Note: If you run into postgres password authentication errors, check out [this blogpost for help](https://juwondaniel.wordpress.com/2016/09/23/solve-mix-ecto-create-postgresql-password-issue-with-phoenix/).
+
+```bash
+mix deps.get
+mix ecto.setup
+sudo apt install npm
+sudo npm install -g yarn
```
Comment (bitboxer) [inline README.md]: you need to call yarn once I think to download all the dependencies, or did that work out of the box?

Comment (berlintam) [inline README.md]: It actually worked out of the box... which in hindsight seems weird. I tried it out just now & ran `yarn` and it did run a script (resolve, fetch, link & build things). But I see no changes. I can always add it in  anyway to be super explicit just incase? wdyt?

Comment (bitboxer) [inline README.md]: Yes, I would love to have it in there, this is the only way to guarantee that the right js files are in the `node_modules` dir I think.

Comment (berlintam) [inline README.md]: done! :)

**Comment by berlintam** [comment]
you're welcome!! :smile:

**Pull request merged**